src/server/clientiface.cpp

```diff
@@ -62,7 +62,8 @@ RemoteClient::RemoteClient() :
 	m_block_optimize_distance(g_settings->getS16("block_send_optimize_distance")),
 	m_block_cull_optimize_distance(g_settings->getS16("block_cull_optimize_distance")),
 	m_max_gen_distance(g_settings->getS16("max_block_generate_distance")),
-	m_occ_cull(g_settings->getBool("server_side_occlusion_culling"))
+	m_occ_cull(g_settings->getBool("server_side_occlusion_culling")),
+	m_perf_tradeoffs(g_settings->getBool("performance_tradeoffs"))
```

Comment:
nitpick: the description of "performance_tradeoffs" talks about visual glitches. maybe adjust it.

Reply:
I think I am just going to remove this part. With the low_priority map edit events it's not needed anymore (or at least much less effective).